packages/augur-artifacts/src/addresses.json

```diff
@@ -110,19 +110,19 @@
  },
  "102": {
   "Augur": "0xFCaf25bF38E7C86612a25ff18CB8e09aB07c9885",
-  "Universe": "0x6a424C1bd008C82191Db24bA1528e60ca92314cA",
-  "LegacyReputationToken": "0xF68374642F47a4C353C78afF17bfA216c930f116",
-  "Cash": "0x25Ff5dc79A7c4e34254ff0f4a19d69E491201DD3",
-  "CancelOrder": "0x6c38ab26F860380a8A2367c79dda3a54aFEf4b1E",
-  "ClaimTradingProceeds": "0xF265D8D30a1A2CdB9857e124010b02765c9a7c70",
-  "CompleteSets": "0x4DdEBCEbE274751dfB129EfC96a588a5242530Ab",
-  "CreateOrder": "0x905419e7A1f96A973AB8988Ba19C84135C6f7122",
-  "FillOrder": "0xce77d3c706e0fe6dA5d71b1249756b5ea0D77202",
-  "Orders": "0x635C8EF61A07dfe49b683D1A34da3547b0d6705E",
-  "ProfitLoss": "0x257b56115544E9519Fb2De5b7b99C3c2150528Fb",
-  "ShareToken": "0x8bDAd8Ee0b47E2e7294A5B8b8B74EA3968f4a177",
-  "SimulateTrade": "0x204cDD1689c8A4da426894C150a0fa672a2eab4C",
-  "Trade": "0xf3bCabd8FAE29F75be271eBE2499EDB4c7C139b7"
+  "Universe": "0x8062dA104239cf70C76B77c61eA988bf6382736a",
```

Comment:
Prob dont need to check in the addresses changes.

Reply:
These are the values on master, no idea why they came up in this PR. I merged with master and double checked. The values are the same.